java/pom.xml

```diff
@@ -1145,6 +1149,9 @@
                     -DCMAKE_INSTALL_LIBDIR=lib/${os.detected.arch}
                     -DCMAKE_INSTALL_PREFIX=java-dist
                     -DCMAKE_UNITY_BUILD=ON
+                    -DPARQUET_BUILD_EXAMPLES=OFF
+                    -DPARQUET_BUILD_EXECUTABLES=OFF
+                    -DPARQUET_REQUIRE_ENCRYPTION=OFF
```

Comment:
FYI: They are not harmful but they are redundant because they are `OFF` by default.